api/operator/v1beta1/common_scrapeparams.go

```diff
@@ -17,6 +17,22 @@ type AttachMetadata struct {
 	Node *bool `json:"node,omitempty"`
 }
 
+type GlobalScrapeConfig struct {
```

Comment:
VMAgent already exposes global scrape configuration params at `spec.` root - `ScrapeInterval`, `ScrapeTimeout` and `ExternalLabels`.

 I'd suggest to add `MetricRelabelConfigs` and `RelabelConfigs` with proper comments.

Reply:
I've copied ScrapeInterval, ScrapeTimeout, ExternalLabels and marked them as deprecated in VMAgentSpec

Reply:
> We cannot simply deprecate field without a good reason
vmagentspec in this case will have RelabelConfig, InlineRelabelConfig and these new RelabelConfigs and MetricRelabelConfigs. Expecting multiple similar issues

Reply:
done